src/gpu/nvidia/cudnn_convolution_impl.hpp

```diff
@@ -248,7 +248,7 @@ struct cudnn_convolution_impl_base_t
         CUDNN_EXECUTE_FUNC_V(cudnnSetConvolutionNdDescriptor, conv_desc,
                 ndims[x] - 2, padding, filter_strides, dilation,
                 cudnnConvolutionMode_t::CUDNN_CROSS_CORRELATION,
-                computation_data_type);
```

Comment:
Can you please elaborate on this change?

Reply:
Because we got `CUDNN_BAD_PARAM` issue for bf16. So we changed it to `CUDNN_DATA_FLOAT`. This resolved the issue.